### PR TITLE
Refactor ETF saving function

### PR DESCRIPTION
**Pull request by larrysammii**
I found digging into the venv's library files to get the JSON file a bit cumbersome, so I quickly did some minor changes and defined a new function that could be optionally imported into user's code. 

When using terminal script, it should remain the same except the JSON file is now stored at pyetf/etfpy.

When imported, the location of the saved file can be specified by the file_path parameter. If no location is specified, the file will be saved to the project root directory.

### CODE REVIEW

**Comment by JakubPluta** [comment]
Hey @larrysammii, thank you for your PR, it was something that I fortget to take care off. 

One thing that I noticed, that the file  etfpy/data/etfs/etfs_list.json is empty. 
I know that you added new functionality to specify where you want to store scraped etfs list, but this file **etfpy/data/etfs/etfs_list.json** is used internally by other code components.
The list of etfs changes very rearly, so I diceded to put this list in this folders, so this way if user want to investigate/analyze single etf first there is a check (if etf ticket exists in this file)
Probably I will add some scheduled job that once a month it scrapes all etfs list and redeployd the package.

So for now, I would suggest to just leave etfpy/data/etfs/etfs_list.json file with the data, and the rest should be okey

File: etfpy/data/etfs/etfs_list.json
Comment (JakubPluta) [inline etfpy/data/etfs/etfs_list.json:1]: Please leave the file with all of the data, as it is used internally

**Review by JakubPluta — CHANGES_REQUESTED** [review]

**Comment by larrysammii** [comment]
Apologies for the error @JakubPluta , I just restored the content within the etfs_list.json.

**Review by JakubPluta — APPROVED** [review]
Everything good now :) Thanks

**Pull request merged**